dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt

```diff
@@ -0,0 +1,76 @@
+package org.dataland.frameworktoolbox.frameworks.eutaxonomynonfinancials.custom
+
+import org.apache.commons.text.StringEscapeUtils
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.intermediate.components.ComponentBase
+import org.dataland.frameworktoolbox.intermediate.components.addStandardCellWithValueGetterFactory
+import org.dataland.frameworktoolbox.intermediate.components.requireDocumentSupportIn
+import org.dataland.frameworktoolbox.intermediate.datapoints.NoDocumentSupport
+import org.dataland.frameworktoolbox.specific.datamodel.TypeReference
+import org.dataland.frameworktoolbox.specific.datamodel.elements.DataClassBuilder
+import org.dataland.frameworktoolbox.specific.fixturegenerator.elements.FixtureSectionBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.UploadCategoryBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+import org.dataland.frameworktoolbox.utils.typescript.TypeScriptImport
+
+/**
+ * Represents the EuTaxonomy-Specific "EuTaxonomyAlignedActivities" component
```

Comment:
Isn't it EuTaxonomyNonFinancials instead of EuTaxonomy. Financials does not use this stuff even if it was devtools integrated, right?

Reply:
This holds for all new files in this folder

Reply:
Ja hast recht.

Also es geht dir darum dass die doc misleading ist richtig?

Dann benenn ich das überall um.  Dann sollte es passen?

Reply:
Jahah. Docs und die Klassennamen. EuTaxonomy ist ja nicht der Framework name.

Reply:
Aber lass es meinetwegen. Ist ja auch irgendwie unnötig. Sieht nie wieder jemand.

Reply:
Hier passt es eigentlich sogar. Bei AmountWithCurrency aber weniger.

Reply:
Ok dann lasse ich es jetzt, wenn es zu verkraften ist. Schließt du die conversation dann?